pkgs/love2d/example/assets/love_example_browser/examples/018_physics_fire_at.lua

```
@@ -0,0 +1,61 @@
+	if love.mouse.isDown(1) then
+		--Sets the starting position of the bullet, this code makes the bullets start in the middle of the player.
+		local startX = StartPos.x + StartPos.width / 2
+		local startY = StartPos.y + StartPos.height / 2
+		
+		local targetX, targetY = love.mouse.getPosition()
+	  
+		--Basic maths and physics, calculates the angle so the code can calculate deltaX and deltaY later.
+		local angle = math.atan2((targetY - startY), (targetX - startX))
+		
+		--Creates a new bullet and appends it to the table we created earlier.
+		newbullet={x=startX,y=startY,angle=angle}
```

Luacheck (1.2.0)

[warning] setting non-standard global variable 'newbullet'

(W111)